src/main/java/com/openlattice/kindling/search/ConductorElasticsearchImpl.java

```diff
@@ -733,7 +733,7 @@ public boolean clearEntitySetData( UUID entitySetId, UUID entityTypeId ) {
         if ( !verifyElasticsearchConnection() ) { return false; }
 
         new DeleteByQueryRequestBuilder( client, DeleteByQueryAction.INSTANCE )
-                .filter( QueryBuilders.termQuery( ENTITY_SET_ID_KEY.toString(), entitySetId.toString() ) )
+                .filter( QueryBuilders.termQuery( ENTITY_SET_ID_FIELD, entitySetId.toString() ) )
```

Comment:
This won't delete the normal entities from linked entity documents, but that's probably fine because they'll be filtered out by permissions restrictions anyway and the linked entity documents will be updated when they're reindexed